pkg/kubelet/apis/config/types.go

```diff
@@ -136,6 +136,9 @@ type KubeletConfiguration struct {
 	// pulls to burst to this number, while still not exceeding registryPullQPS.
 	// Only used if registryPullQPS > 0.
 	RegistryBurst int32
+	// rootDirectory is the directory path to place kubelet files (volume
+	// mounts,etc).
+	RootDirectory string
```

Comment:
we already have VolumePluginDir and the existing flag is root-dir... suggest `RootDir`